### ISSUE TITLE
Terminals scrolling problem (needs resizing)

### ISSUE THREAD

**Issue by yogeek** [play-with-docker/play-with-docker.github.io#105]
There are some displaying problems with terminal scroll bars in some labs. 
Example to reproduce the problem :

- open [https://training.play-with-docker.com/ops-s1-swarm-intro/](https://training.play-with-docker.com/ops-s1-swarm-intro/) lab with Chrome or Firefox
- execute the first command of the lab `docker swarm init --advertise-addr $(hostname -i)` in the upper terminal (the scroll bar doen not follow the output)
- scroll down the first terminal and try to enter a character on the command line
=> the scroll bar goes back up and prevents to see what is typed

It seems that resizing the page or moving the central vertical slide bar to change left-right proportion once makes the scrolling issue disappear. 

OS : Ubuntu 16.04.4 LTS
Chrome version tested : Version 64.0.3282.186 (Build officiel) (64 bits)
Firefox version tested : 58.0.2 (64 bits)

Thank you !

**Comment by marcosnils** [comment]
> It seems that resizing the page or moving the central vertical slide bar to change left-right proportion once makes the scrolling issue disappear.

Yes, we we're aware that there are some issues with the terminals the first time they render. That's why we added this message:

![image](https://user-images.githubusercontent.com/1578458/36948222-68cb0110-1fb6-11e8-82f6-0b515d45a1b3.png)

It's pretty tricky yo make the terminal scrolls work ok in all browsers and platforms. This shouldn't happen using chrome though.

Which OS are you using?, can you reproduce it consistently?. Have you tried a different browser?

**Comment by yogeek** [comment]
hello @marcosnils  
Oh ok, indeed I have seen the warning message but misinterpreted it : I thought it only concerned the case when the terminal window does not show the command line at all. Here it is only after typing some commands that the issue appeared and as all was ok at the beginning, I forgot about the message... Sorry for the misinterpretation.

My OS and browser versions are mentioned in the issue description and yes, with these environments, I can reproduce it each time I refresh the page to start the lab again.

Thanks for your help

**Comment by marcosnils** [comment]
Sorry, I didn't see the OS. I'll give it a shot during the week and see if I can reproduce

**Comment by marcosnils** [comment]
@yogeek can you try using chrome to see if the problem is there as well?

**Comment by yogeek** [comment]
> OS : Ubuntu 16.04.4 LTS
> Chrome version tested : Version 64.0.3282.186 (Build officiel) (64 bits)
> Firefox version tested : 58.0.2 (64 bits)

@marcosnils I already tested with both firefox and chrome before submitting but only with these 2 specific versions

**Comment by idkjs** [comment]
Same issue here. Firefox and chrome.
```
PrisBook:~$ macnodeyarnv
ProductName:	Mac OS X
ProductVersion:	10.13.4
BuildVersion:	17E202
yarn version: 1.6.0
node version: v9.5.0
```

Firefox
![screen shot 2018-05-04 at 1 02 17 pm](https://user-images.githubusercontent.com/2370391/39624927-79b33a78-4f9b-11e8-9d87-1ba6dd1c0ebb.png)

Chrome
<img width="638" alt="screen shot 2018-05-04 at 1 01 25 pm" src="https://user-images.githubusercontent.com/2370391/39624933-8365e61a-4f9b-11e8-8664-0ab25090c97a.png">

**Comment by TjrGithub** [comment]
I also have this issue on Ubuntu 19.10, latest Firefox 74.0. Whenever I type something, the terminal scrolls fully up and hides anything I type.
